spec/features/user_can_reserve_an_appointment_spec.rb

```diff
@@ -1,16 +1,24 @@
 require "rails_helper"
 
-feature "user can go to reserve an appointment from home page" do
-  scenario "sees the button to reserve appointment" do
-    visit root_path
-
-    expect(page).to have_link("Solicitar un turno")
-  end
-
+feature "user can reserve an appointment from home page" do
   scenario "without being logged in" do
+    skip
+
+    car = create(:car)
     visit root_path
     click_link("Solicitar un turno")
 
     expect(current_path).to eq(new_appointment_path)
+    # expect(page).to have_content("Reserva de turnos")
+
+    # select(Date.tomorrow, from: "Seleccione una fecha")
+    # select(Time.now.hour, from: "Seleccione un horario")
+
+    fill_in "Seleccione una fecha", with: Date.tomorrow
+    fill_in "Seleccione un horario", with: Time.now.hour
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.